state/sqlserver/migration.go

```diff
@@ -248,36 +244,12 @@ func (m *migration) ensureTypeExists(ctx context.Context, db *sql.DB, mr migrati
 	return runCommand(ctx, db, tsql)
 }
 
-/* #nosec. */
-func (m *migration) ensureBulkDeleteStoredProcedureExists(ctx context.Context, db *sql.DB, mr migrationResult) error {
```

Comment:
Why this is not needed?

Reply:
Removed native BulkDelete from this component. Otherwise that was behaving atomically, which is what we wanted to get rid of.